hits_cpp/loadSequences.cpp

```diff
@@ -7,7 +7,7 @@
 #include "trim.hpp"
 
 using namespace std;
-SequencesMap loadSequences(string faaPath, bool verbose) {
+SequencesMap loadSequences(string faaPath, float *factor, bool verbose) {
```

Comment:
Better use reference (&) instead of pointer (*) for memory safety,
or expose factor as function (getRpmFactor) instead of returning it as part of this function (clearer code/separation of concerns)